internal/cli/atlas/cloudprovider/cloud_provider.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudprovider
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/atlas/cloudprovider/accessroles"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "cloudProvider"
```

Comment:
[nit] sorry I missed this one in the last review but this should also be plural, same with the package name

Reply:
can I merge and change it in the next pr, please?